### PR TITLE
Better icons and non-deprecated url

### PR DESCRIPTION
**Pull request by lakinwecker**

### CODE REVIEW

File: heltour/tournament/templates/tournament/team_pairings.html [hunk 1]
```diff
@@ -110,8 +110,8 @@ <h3>Pairings</h3>
 					<tr>
 						<td class="pairing-player player-{{ pairing.white_team_color }}">
 							{% if pairing.white_team_player %}
-							<a href="https://lichess.org/games/export/{{ pairing.white_team_player.lichess_username }}?perfType=classical,rapid&color={{ pairing.white_team_color }}" target="_blank" rel="nofollow">
-								<img src="{% static "tournament/img/book-pile.png" %}">
+							<a class="download-games" href="https://lichess.org/api/games/use/{{ pairing.white_team_player.lichess_username }}?perfType={{ league.rating_type }}&color={{ pairing.white_team_color }}" target="_blank" rel="nofollow">
```
Comment (cyanfish) [inline heltour/tournament/templates/tournament/team_pairings.html]: Typo here

Comment (lakinwecker) [reply heltour/tournament/templates/tournament/team_pairings.html]: Fixed.

Comment (lakinwecker) [reply heltour/tournament/templates/tournament/team_pairings.html]: Damnit, my commit didn't go through

Comment (lakinwecker) [reply heltour/tournament/templates/tournament/team_pairings.html]: Now it's really fixed.

**Pull request merged**